node/node.go

```diff
@@ -1578,11 +1582,24 @@ func (app *App) grpcService(svc grpcserver.Service, lg log.Log) (grpcserver.Serv
 		service := v2alpha1.NewAccountService(app.db, app.conState)
 		app.grpcServices[svc] = service
 		return service, nil
+	case v2alpha1.SmeshingIdentities:
+		nodeIds := make(map[types.NodeID]struct{})
+		for _, signer := range app.signers {
+			nodeIds[signer.NodeID()] = struct{}{}
+		}
+
+		configuredPoets := make(map[string]struct{})
+		for _, server := range app.Config.PoetServers {
+			configuredPoets[server.Address] = struct{}{}
+		}
+
+		service := v2alpha1.NewSmeshingIdentitiesService(app.db, configuredPoets, app.atxBuilder, nodeIds)
+		app.grpcServices[svc] = service
```

Comment:
I feel like this doesn't need to be its own endpoint? Would it make sense to add it to the `v2alpha1.Activation` service?

Reply:
I thought may be it is better to combine this and `post_info_service` to one service. Because there we return states of post per identity and here is poet per identity. 
But PostInfoService is grpcserver package, so..

Reply:
@poszu what's your take on this? Shall we leave it as it is, add it to the activation endpoint or add it to the `PostInfo` service? What about `v2alpha1`?